DirectProgramming/DPC++FPGA/PlatformTests/board_test/README.md

```diff
@@ -0,0 +1,341 @@
+# Board Test
+This design contains tests to check FPGA board interfaces and reports the following metrics:
+1. Host to device global memory interface bandwidth
+2. Kernel clock frequency
+3. Kernel launch latency
+4. Kernel to device global memory bandwidth
+
+***Documentation***:  The [DPC++ FPGA Code Samples Guide](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html) helps you to navigate the samples and build your knowledge of DPC++ for FPGA. <br>
+The [oneAPI DPC++ FPGA Optimization Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide) is the reference manual for targeting FPGAs through DPC++. <br>
+The [oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general resource for target-independent DPC++ programming.
+
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04/20.04, RHEL*/CentOS* 8, SUSE* 15
```

Comment:
why no windows support? what if someone wants to develop a windows platform?

Reply:
Adding Windows support